lib/transport/transport-factory-socket.c

```diff
@@ -27,7 +27,7 @@
 
 typedef struct _TransportFactorySocket TransportFactorySocket;
 
-DEFINE_TRANSPORT_FACTORY_ID_FUN(transport_factory_socket_id);
+DEFINE_TRANSPORT_FACTORY_ID_FUN(socket, transport_factory_socket_id);
```

Comment:
I think it would be more understandable if we passed the name of the transport as a string.

For example:
```C
DEFINE_TRANSPORT_FACTORY_ID_FUN("socket", transport_factory_socket_id);
```

Reply:
Sorry, already fixed (`transport: use a sequence as uniq id instead of location macros `)